packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/helpers/fetchRecentSubjects/fetchTalkData.js

```diff
@@ -1,13 +1,15 @@
-import talkClient from 'panoptes-client/lib/talk-client'
+import { panoptes } from '@zooniverse/panoptes-js'
 
 async function fetchTalkData (projectId) {
-  return talkClient.type('comments')
-    .get({
-      focus_type: 'Subject',
-      page_size: 10,
-      section: `project-${projectId}`,
-      sort: '-created_at'
-    })
+  const query = {
+    focus_type: 'Subject',
+    page_size: 10,
+    section: `project-${projectId}`,
+    sort: '-created_at'
+  }
+
+  return panoptes.get('/comments', query, {}, process.env.TALK_HOST)
```

Comment:
Non-blocking: I didn't think of this when suggesting it, but this is not respecting the query param env setting. I don't think it's important for now, but just something to keep in mind.

Reply:
I think you can run `PANOPTES_ENV=production yarn dev` to develop against the production API.